book/src/testnet-replicator.md

```diff
@@ -0,0 +1,200 @@
+## Testnet Replicator
+This document describes how to setup a replicator in the testnet
+
+Please note some of the information and instructions described here may change
+in future releases.
+
+### Overview
+Replicators are specialized light clients. They download a part of the
+ledger (a.k.a Segment) and store it. They earn rewards for storing segments.
+
+The testnet features a validator running at testnet.solana.com, which
+serves as the entrypoint to the cluster for your replicator node.
+
+Additionally there is a blockexplorer available at
+[http://testnet.solana.com/](http://testnet.solana.com/).
+
+The testnet is configured to reset the ledger daily, or sooner
```

Comment:
Probably should mention somewhere that when the testnet restarts the user needs to recreate their storage accounts

Reply:
Oh yes, Ill add that.